service/src/main/java/bio/terra/catalog/common/DaoKeyHolder.java

```diff
@@ -31,10 +28,9 @@ public String getString(String fieldName) {
 
   public <T> T getField(String fieldName, Class<T> type) {
     Map<String, Object> keys = getKeys();
-    if (keys != null) {
-      Object fieldObject = keys.get(fieldName);
-      return type.cast(fieldObject);
-    }
-    return null;
+    // In practice getKeys() will never return null
+    @SuppressWarnings("ConstantConditions")
+    Object fieldObject = keys.get(fieldName);
```

Comment:
If we aren't null checking, could we inline `getKeys()`?

Reply:
what does inline mean?

Reply:
Instead of doing:
```
Map<String, Object> keys = getKeys();
Object fieldObject = keys.get(fieldName);
```
Do:
```
Object fieldObject = getKeys().get(fieldName);
```